wiki/Tournaments/SS_Tournament/2018/en.md

```diff
@@ -64,180 +95,235 @@ Blue BAN Red BAN  Blue BAN Red BAN : Red MAP Blue MAP Red MAP Blue MAP …
 ![Qualifiers Mappool](Qaulifiers.jpg)
```

Comment:
why not use new website links for those, when you were using new website links for profiles ?